setup.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+try:
+    from setuptools import setup, find_packages
+except ImportError:
+    from distutils.core import setup
+
+
+with open('README.rst') as readme_file:
+    readme = readme_file.read()
+
+with open('requirements.txt') as req_file:
+    requirements = req_file.read().split('\n')
+
+with open('requirements-dev.txt') as req_file:
+    requirements_dev = req_file.read().split('\n')
```

Comment:
These blocks will throw exceptions if the files they're trying to open don't exist.

It's also probably a better idea to list the dependencies explicitly instead of reading them from `requirements.txt`. (Requirements files allow all sorts of things that `setup.py` doesn't, and you'll need to filter out empty lines and comments even if the requirements files are only using basic features.)